apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/injectable/PropertiesHolder.java

```diff
@@ -161,11 +161,19 @@ public String toString() {
 
     public String dumpAllProperties() {
         final StringBuffer sb = new StringBuffer("PropertiesHolder_DUMP : \n");
-        properties.forEach((k, v) ->
-            sb.append('\'').append(k).append("'->").append(v).append(", ")
+        properties.forEach((k, v) -> {
```

Comment:
Does not make sense to secure admin password here since ServerInfoController already do that